knowledge-base/editor-add-icons.md

```diff
@@ -0,0 +1,197 @@
+        var head = doc.querySelector("head");
+
+        var cssLink1 = document.createElement("link");
+        cssLink1.href = "https://cdnjs.cloudflare.com/ajax/libs/open-iconic/1.1.1/font/css/open-iconic-bootstrap.min.css";
+        cssLink1.rel = "stylesheet";
+        cssLink1.type = "text/css";
+
+        var cssLink2 = document.createElement("link");
+        cssLink2.href = "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css";
+        cssLink2.rel = "stylesheet";
+        cssLink2.type = "text/css";
+
+        head.appendChild(cssLink1);
+        head.appendChild(cssLink2);
+    };
+</script>
+````
+
+### Add icons in Div Edit Mode
+
+When the [EditMode of the Editor is set to `EditorEditMode.Div`]({%slug editor-edit-modes-div%}), the content area is an editable <div> element that inherits the CSS rules from the current page.
+
+This allows you to include the icons stylesheet in the `<head>` of the web page along with the other stylesheets. 
+
+>caption Add icons in an Editor with Div Edit Mode
+
+````CSHTML
+@using Telerik.Blazor.Components.Editor
+
+@* Just one example of including custom font icon libraries.
+Make sure to use the correct way and resources for your actual project *@
+<link href="https://cdnjs.cloudflare.com/ajax/libs/open-iconic/1.1.1/font/css/open-iconic-bootstrap.min.css" rel="stylesheet" />
+<link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" rel="stylesheet" />
+
+<TelerikEditor @bind-Value="@EditorValue"
+               Tools="@EditorTools"
+               Schema="schemaProvider"
+               EditMode="@EditorEditMode.Div"
+               Width="600px"
+               Height="300px">
+</TelerikEditor>
+
+@code {
+    private string EditorValue { get; set; } = @"Here is an example icon in the Editor content <i class='fa fa-info-circle'></i>";
+
+    private List<IEditorTool> EditorTools { get; set; } = new List<IEditorTool>() { new ViewHtml() };
+}
+
+@* Move JavaScript code to a separate JS file in production *@
+<script suppress-error="BL9992">
+    var icon = {
+        attrs: {
+            class: { default: null },
+            type: { default: null },
+        },
+        group: "inline",
+        content: "text*",
+        inline: true,
+        parseDOM: [
+            {
+                tag: "i",
+                getAttrs: (dom) => ({
+                    class: dom.getAttribute("class"),
+                }),
+            },
+        ],
+        toDOM: (node) => {
+            const attrs = {
+                class: node.attrs.class
+            };
+            return ["i", attrs];
+        },
+    };
+    debugger
+
+    window.schemaProvider = (args) => {
+        const schema = args.getSchema();
+        const Schema = args.ProseMirror.Schema
+
+        let nodes = schema.spec.nodes.addToEnd("i", icon);
+
+        const newSchema = new Schema({ nodes });
+        return newSchema;
+    }
+
+</script>
+```
```

Comment:
Consider adding a See Also section with a few relevant links.

Reply:
See https://github.com/telerik/blazor-docs/pull/2297#discussion_r1704107336

Reply:
Some scenarios may require users to add icons and not just see them. So, you can link here to the Custom Tools article, which is not mentioned in the KB itself.